.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate Theme Entry
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - community-css-themes.json
+
+jobs:
+  theme-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "refs/pull/${{ github.event.number }}/merge"
+      - uses: actions/setup-node@v2
+      - run: npm install probe-image-size
```

Comment:
Is there a simple image size script we can embed directly into the workflow so that we don't need to npm install here? This means that it will run npm install for each execution which I suspect will slow down the script a lot. Obviously it's not going to be hugely detrimental (current execution is ~5s E2E), but still would be nice to avoid.